uamqp/message.py

```diff
@@ -596,7 +596,7 @@ def gather(self):
             try:
                 if not data.application_properties and self.application_properties:  # Message-like object
                     data.application_properties = self.application_properties
-                message_bytes = data.encode_message()
+                message_bytes = data._encode_message()  # pylint: disable=protected-access
```

Comment:
We need to support both `encode_message` and `_encode_message`

Reply:
you're right, otherwise we lose compatibility.